src/domain/bitcoin/lightning/invoice-expiration.ts

```diff
@@ -1,16 +1,15 @@
-import moment from "moment"
+import { toSeconds } from "@domain/primitives"
 
 const DEFAULT_EXPIRATIONS = {
-  BTC: { value: 1, unit: "days" as moment.unitOfTime.DurationConstructor },
-  USD: { value: 2, unit: "minutes" as moment.unitOfTime.DurationConstructor },
+  BTC: { delay: toSeconds(60 * 60 * 24) },
+  USD: { delay: toSeconds(2 * 60) },
```

Comment:
We have a section with constants [here](https://github.com/GaloyMoney/galoy/blob/182fe1f816c738f222d8a5ae8864931a2ad9c2bc/src/config/app.ts#L35) in `app.ts` that we can probably add some of these values to
```suggestion
  BTC: { delay: toSeconds(SECS_PER_DAY) },
  USD: { delay: toSeconds(SECS_PER_2_MINS) },
```